tinkerers/Vivek_Joseph.md

```diff
@@ -0,0 +1,11 @@
+# INTRODUCTION
+
+My name is **Vivek V Joseph**
+I am studying **CSE SEM_1** at **CUSAT**
+_I love to do_
+- Draw
+- game
+- play basketball
+![Image_alt_text](C:\Users\Vichu\Desktop\peace iout meme.jpg)
```

Comment:
Heyy if you want to include images in your file.. use the URL convention mentioned in the README of the repo with your file name. Also try not to include any spaces in your file name, it gets messed when you reference file paths.